frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
```

Comment:
Where? Home screen
Issue: Button shall be full width in the mat card (check current prod for reference)

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/c9d7bf24-75f0-4a47-a4e9-db069774f496)